devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java

```diff
@@ -149,6 +177,7 @@ public void buildQuarkus() {
                 .setAppArtifact(appModel.getAppArtifact())
                 .setLocalProjectDiscovery(false)
                 .setIsolateDeployment(true)
+                .setForcedDependencies(forcedDependencies)
```

Comment:
If you are setting an `existingModel` then calling `setForcedDependencies(deps)` here won't have any affect, given that the model has already been resolved.

Reply:
So, this is redundant given that I also pass the forced dependencies in the `resolveManagedModel`. I'll remove it.